app/index.js

```diff
@@ -48,9 +48,14 @@ app.post('/',
             path: req.file.path,
             name: req.file.originalname
         };
+        var options = {
+            checkVersion: req.body.version || 'v2'
```

Comment:
Alternative naming: "latest"
By that we always have "latest" and "--v1" or "latest" and "--v2" etc.

Reply:
I was also thinking of adding a latest and previous property to the config to grab them from there. Does that make sense?

Or would it be better to have only the previous version fixed?

Reply:
> I was also thinking of adding a latest and previous property to the config to grab them from there. Does that make sense?

I wouldn't go that far yet, because if we release Ghost 3.0, we have to put manual work into GScan anyway. I don't imagine, we can just simply reconfigure the config and it will work.

Reply:
But it would be nice to have it globally defined somewhere, so we don't need to change it in x places.

Reply:
> But it would be nice to have it globally defined somewhere, so we don't need to change it in x places.

Yeah, that was my intention behind this. Will look where we can do this.

Reply:
Cool :)